pages/level/[[...params]].tsx

```diff
@@ -59,15 +58,26 @@ export async function getStaticPaths() {
 }
 
 interface LevelParams extends ParsedUrlQuery {
-  id: string;
+  params: string;
 }
 
 export async function getStaticProps(context: GetServerSidePropsContext) {
   await dbConnect();
 
-  const { id } = context.params as LevelParams;
-  const level = await LevelModel.findById<Level>(id)
-    .populate('userId', 'name');
+  const { params } = context.params as LevelParams;
```

Comment:
Good question... I was thinking it could be good to support both at least for short while to help debug in case we have future issues - but it may add complexities. What do you think?

Reply:
I think it's not too complex but you're right there may be bugs at first. Maybe we can support both for a bit and then if everything looks good in production we can get rid of level/id later on.